src/utilities/linkHighlightedJson.js

```diff
@@ -0,0 +1,83 @@
+import _ from 'lodash';
+import {xdrViewer, singleAccount} from './linkBuilder';
+
+// Map of functions specific to JSON property keys that creates urls for a given
+// value.
+// A function can also return undefined if it doesn't deem the value worthy of
+// linking (for example: invalid account ID or keys that have ambiguous names)
+let accountIdGenerator = (value) => {
+  if (value.length === 56) {
+    return singleAccount(value);
+  }
+};
+let valueUrlGenerator = {
+  href: (value) => {
+    return value;
+  },
+  'envelope_xdr': (value) => {
+    return xdrViewer(value, 'TransactionEnvelope');
+  },
+  'result_xdr': (value) => {
+    return xdrViewer(value, 'TransactionResult');
+  },
+  'result_meta_xdr': (value) => {
+    return xdrViewer(value, 'TransactionMeta');
+  },
+  'id': accountIdGenerator,
+  'public_key': accountIdGenerator,
+  'account_id': accountIdGenerator,
+  'funder': accountIdGenerator,
+  'account': accountIdGenerator,
+  'source_account': accountIdGenerator,
+  'destination_account': accountIdGenerator,
+};
+
+// linkHighlightedJson annotates and adds relevant links to json code highlighted
+// by prism.js.
+
+// @param {string} code - Valid html generated by prism.js in JSON language mode
+export default function linkHighlightedJson(code) {
+  let parser = new DOMParser();
+  let document = parser.parseFromString(code, 'text/html');
+  let body = document.childNodes[0].childNodes[1];
+
+  // Edit the token elements in place
+  _.each(body.children, (token, index) => {
+    if (token.className.indexOf('property') > 0) {
+      annotatePropertyToken(token);
+    }
+  });
+  return body.innerHTML;
+}
+
+// Takes in a property highlighted item. If functionality is defined for it's
+// corresponding property (only string type supported) and the content is valid,
+// then we will turn the value node into a link.
+// @param {HTMLElement} propertyToken - The token element of property type (assigned by prism)
+function annotatePropertyToken(propertyToken) {
+  let valueToken = propertyToken.nextElementSibling.nextElementSibling;
```

Comment:
I could look at the `span.token.property` and find it's closest next neighbor that is `span.token.string`. But some times it might not be a string and we would have to do some validations.

In correctly formed JSON, there should always be a value that follows the property. If I wanted to decouple things, I could iterate over next tags/nodes until I find one with text that is not `:` or ` `. This way, the amount of next we go is based on content in tags and not what Prism gives us.

I'm not sure how to prevent breaking changes from Prism. Someone would have to be manually updating Prism because I pasted it into a js file in the laboratory. The in this case of a manual upgrade, someone will see the errors when upgrading.

Is this the right approach?